site/_scss/main.scss

```diff
@@ -70,7 +70,6 @@
 @import 'blocks/logo';
 @import 'blocks/devtools';
 @import 'blocks/select-loader';
-@import 'blocks/language-list';
```

Comment:
Please remove "blocks/_dates". It just had one class that set a color, which we now do with utility CSS.